crates/context/config/Cargo.toml

```diff
@@ -82,7 +85,8 @@ stellar_client = [
 
 evm = [
```

Comment:
should this be `ethereum` instead?

Reply:
https://github.com/calimero-network/core/pull/1142#discussion_r1995920234

we'll revise the name in a separate PR